02week/pigLatin.js

```diff
@@ -8,13 +8,36 @@ const rl = readline.createInterface({
 });
 
 
-function pigLatin(word) {
-
-  // Your code here
+function pigLatin(input) {
+  let vowIndex = -1;
+  const word = input.split('');
```

Comment:
Your game is working except for trimming whitespace. Find the method on strings to take care of this task. Also, you create a variable called word on 13 but don't use it in the if statement below. If you .lowerCase before putting it in the variable you can be sure you're comparing everything to one case: the variable you created.